sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -429,6 +429,7 @@ public bool IsRunning
 
             ProcessingReadEventOptions = new ReadEventOptions
             {
+                OwnerLevel = 0,
```

Comment:
>OwnerLevel = 0, [](start = 16, length = 15)

Stealing receiver requires higher owner level than the current receiver's. Is this change taking that into account?

Reply:
There's no current use of owner level-based stealing in the current load balancing algorithm.   Unless you've got another scenario in mind that I'm overlooking, I think we're good here.